website/quiz.py

```diff
@@ -280,6 +303,26 @@ def quiz_feedback(level_source, question_nr):
                                lang=g.lang)
 
 
+def get_result_items(quiz_answers, questions):
+
+    result_items = []
+    print(questions)
+    for i in range(len(questions)):
```

Comment:
Can't we iterate the questions itself instead of using a loop counter. So simply `for question in questions`.